spec/features/issues_spec.rb

```diff
@@ -287,7 +288,7 @@
         sleep 2 # wait for ajax stuff to complete
         first('.user-result').click
 
-        page.should have_content "Assignee: Unassigned"
+        page.should have_content "Assignee: none"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.